mltsp/Flask/static/js/scripts.js

```diff
@@ -1828,3 +1822,50 @@ function drawScatterplotMatrix(datafilename){
     });
 
 }
+
+
+
+function generate_pred_results_table(div_id, results_dict){
```

Comment:
Yes, you're right of course.  This is why we need components that can
render themselves from data! (Which is, in essence, exactly what you
coded up here.)

Here is an alternative hack:

http://stackoverflow.com/questions/28492221/dynamically-render-template-without-reloading-page

But, ultimately, this is where we should start thinking about ReactJS:

https://facebook.github.io/react/